parm/config/gfs/config.wave

```diff
@@ -27,49 +27,52 @@ export CDUMPRSTwave="gdas"
 # Grids for wave model
 export waveGRD=${waveGRD:-'mx025'}
 
+
+#grid dependent variable defaults 
+export waveGRDN=${waveGRDN:-'1'} # gridnumber for ww3_multi
+export waveGRDG=${waveGRDG:-'10'} # gridgroup for ww3_multi
+export USE_WAV_RMP=${USE_WAV_RMP:-'NO'} #yes/no rmp grid remapping pre-processed coefficients
+export waveMULTIGRID=${waveMULTIGRID:-'.false.'} #true/false for multi or shel 
+export MESH_WAV=${MESH_WAV:-"mesh.${waveGRD}.nc"}  #Mesh grid for wave model for CMEPS
+
+export waveesmfGRD=${waveesmfGRD:-' '} # input grid for multigrid 
```

Comment:
No longer need the passthroughs

Also, the comments say the grid numbers are for ww3_multi, but waveMULTIGRID is false. Is this a mismatch in the defaults?

```suggestion
#grid dependent variable defaults 
export waveGRDN='1' # gridnumber for ww3_multi
export waveGRDG='10' # gridgroup for ww3_multi
export USE_WAV_RMP='NO' #yes/no rmp grid remapping pre-processed coefficients
export waveMULTIGRID='.false.' #true/false for multi or shel
export MESH_WAV="mesh.${waveGRD}.nc" #Mesh grid for wave model for CMEPS

export waveesmfGRD=' ' # input grid for multigrid
```

Reply:
The pass throughs are helpful if you want to define these values elsewhere like in the deafults.s2s.  I'd like to keep them but do not want to further hold up the GEFS group either who really needs this PR.  

Indeed multigrid is false so theoretically these don't really "have" to be set, but they have default values for completeness here.

Reply:
defaults.s2s is going to go away soon.

My question was more the mismatch between this comment:
`# gridnumber for ww3_multi`
and
`export waveMULTIGRID='.false.' #true/false for multi or shel`

If the default grid number is for ww3_multi, why is waveMULTIGRID false? Something does seem to match here.

Reply:
the variable waveGRDN is only used if waveMULTIGRID=true.  However, you can run multi-grid with a single grid.   I have set these to values that would be appropriate for a single grid if you had waveMULTIGRID=true.  So basically if you turned waveMULTIGRID=true, everything would be set for you to run.   However, we're using waveMULTIGRID=false.  Right now the workflow still supports the option of waveMULTIGRID=true, so now we have the grid specific variables set in case we run w/multi grid.   To remove multigrid support in my opinion would be out of the scope of this PR, but just let me know what needs to be done.

Reply:
Okay, I understand now.

Reply:
@WalterKolczynski-NOAA what additional changes are required for this PR?  Do I need to revert the passthroughs or can they be left as is?